UIKitTemplate/UIKitTemplate/StackViewController.swift

```diff
@@ -0,0 +1,81 @@
+// StackViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Сфетофор на стек вью
+final class StackViewController: UIViewController {
+    // MARK: - Visual Components
+
+    private var stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.spacing = 10
+        stackView.axis = .vertical
+        stackView.distribution = .fillEqually
+        return stackView
+    }()
+
+    private var redSquareView: UIView = {
```

Comment:
```suggestion
    private var redSquareView: UIView = {
```
тоже самое 0 смысла, ты не меняешь вьюшку, а даже если ты меняешь какое то ее свойство потом - то необязательно ее делать var
меняй на вар только тогда когда тебе что то мешает и ругается Xcode